x-pack/legacy/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/WatcherFlyout.tsx

```diff
@@ -83,7 +83,8 @@ export class WatcherFlyout extends Component<
   WatcherFlyoutProps,
   WatcherFlyoutState
 > {
-  static contextType = CoreContext;
+  static contextType = KibanaCoreContext;
+  context!: React.ContextType<typeof KibanaCoreContext>;
```

Comment:
AFAICT this is the way to let TS know what type `this.context` should be in the rest of the class, so that each reference to `this.context` doesn't need to self-type.